SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -41,7 +41,7 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 	FORCEINLINE bool IsReadyForReplication()
 	{
 		// Make sure we have authority
-		if (Actor->Role != ROLE_Authority)
+		if (Actor->Role != ROLE_Authority && !bCreatingNewEntity)
```

Comment:
how come? (curiosity)

Reply:
This is needed because the role could be flipped if spawning on a worker that cannot have authority. This change allows us to create a new entity and do the initial replication even in this case.